initializers/ip_addresses.yml

```diff
@@ -0,0 +1,18 @@
+
+#- address: 10.1.1.1/24
+#  device: router1
+#  family: 4
```

Comment:
Does `family` need to be defined explicitly? I believe Netbox can figure it out from the IP you give it.